lib/utils.js

```diff
@@ -177,6 +177,30 @@ function parseStack(err, cb) {
   });
 }
 
+function fetchGitSha(basePath, cb) {
+  var headPath = path.resolve(basePath, '.git', 'HEAD');
+
+  // just let it return an err if the file does not exist
+  fs.readFile(headPath, 'utf8', function(err1, headFileData) {
+    if (err1) {
+      return cb(err1);
+    }
+    if (headFileData.indexOf('ref: ') === 0) {
+      var ref = headFileData.split(' ')[1].trim();
+      var refPath = path.resolve(basePath, '.git', ref);
+      fs.readFile(refPath, 'utf8', function(err2, refFileData) {
+        if (err1) {
```

Comment:
eslint will complain about it. And yeah, I thought I forcepushed those along with the singlequote, just to realize it's still stuck at entering my passphrase :)

Give me a sec and I'll get a commit to fix those.